src/sysctl.cr

```diff
@@ -0,0 +1,87 @@
+lib LibC
```

Comment:
Yes. The bindings in those folders should actually be automatically generated from the `.h` files on each platform, but I never got [the generator](https://github.com/ysbaddaden/posix) to work. @ysbaddaden?